client/jetpack-cloud/sections/partner-portal/credit-card-fields/set-as-primary-payment-method.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+import { CheckboxControl } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { useDispatch } from 'react-redux';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+const CheckboxWrapper = styled.div`
+	margin-top: 16px;
+`;
```

Comment:
Just out of curiosity (maybe this is a question for someone who has worked with emotion before), what benefits does it bring to use emotion to style that div over a CSS class.

Reply:
I’m not familiar with the pros and cons of both approaches, but looking at [the documentation](https://emotion.sh/docs/styled) I can see that this package support some neat features.